src/datadog/config_manager.h

```diff
@@ -34,6 +37,8 @@ class ConfigManager {
   // Return the `SpanDefaults` consistent with the most recent configuration.
   std::shared_ptr<const SpanDefaults> get_span_defaults();
 
+  bool get_report_traces();
```

Comment:
- [ ] document (and rename?)

Reply:
We could follow the lead of `TracerConfig` and omit the `get_` prefix. It's fine either way.

Yes, please do add a comment here. Maybe:
```
// Return whether spans should be sent to the collector.
```
I don't like the use of the word "should." Something like that, anyway.